aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md

```diff
@@ -1,202 +1,196 @@
-
+> In JavaScript, the reference to the server class and its members is in camelCase. The code sample references the C# `ChatHub` class in JavaScript as `chatHub`.
 
-The following code shows how to create a callback function in the script. The hub class on the server calls this function to push content updates to each client. The optional call to the `htmlEncode` function shows a way to HTML encode the message content before displaying it in the page, as a way to prevent script injection.
+In this code block, you create a callback function in the script.
 
 [!code-html[Main](tutorial-getting-started-with-signalr-and-mvc/samples/sample7.html)]
 
-The following code shows how to open a connection with the hub. The code starts the connection and then passes it a function to handle the click event on the **Send** button in the Chat page.
-
-> [!NOTE]
-> This approach ensures that the connection is established before the event handler executes.
+The hub class on the server calls this function to push content updates to each client. The optional call to the `htmlEncode` function shows a way to HTML encode the message content before displaying it in the page. It's a way to prevent script injection.
 
+This code opens a connection with the hub.
 
 [!code-javascript[Main](tutorial-getting-started-with-signalr-and-mvc/samples/sample8.js)]
 
-<a id="next"></a>
+> [!NOTE]
+> This approach ensures that you establish a connection before the event handler executes.
 
-## Next Steps
+The code starts the connection and then passes it a function to handle the click event on the **Send** button in the Chat page.
 
-You learned that SignalR is a framework for building real-time web applications. You also learned several SignalR development tasks: how to add SignalR to an ASP.NET application, how to create a hub class, and how to send and receive messages from the hub.
+## Additional resources
 
 For a walkthrough on how to deploy the sample SignalR application to Azure, see [Using SignalR with Web Apps in Azure App Service](../deployment/using-signalr-with-azure-web-sites.md). For detailed information about how to deploy a Visual Studio web project to a Windows Azure Web Site, see [Create an ASP.NET web app in Azure App Service](https://azure.microsoft.com/documentation/articles/web-sites-dotnet-get-started/).
 
-To learn more advanced SignalR developments concepts, visit the following sites for SignalR source code and resources :
+To learn more advanced SignalR developments concepts, go to these sites for SignalR source code and resources:
+
+* [SignalR Project](http://signalr.net)
+
+* [SignalR GitHub and Samples](https://github.com/SignalR/SignalR)
+
+* [SignalR Wiki](https://github.com/SignalR/SignalR/wiki)
+
+## Next Steps
+
+In this tutorial you learned how to:
```

Comment:
comma after tutorial

change to follow example of changes in the intro version of this list, i.e, "in this tutorial, you:" and put the verb in past tense in each bullet